docs/user/openstack/README.md

```diff
@@ -287,6 +294,35 @@ Alternatively, if you don't control the DNS, you can add the hostnames in `/etc/
 <apps FIP> <app name>.apps.<cluster name>.<base domain>
 ```
 
+**WARNING:** *this workaround will make the API accessible only to the computer with these `/etc/hosts` entries. This is fine for your own testing (and it is enough for the installation to succeed), but it is not enough for a production deployment. In addition, if you create new OpenShift apps or routes, you will have to add their entries too.*
+
+##### External API Access
+
+If you have specified the API floating IP (either via the installer prompt or by adding the `lbFloatingIP` entry in your `install-config.yaml`) the installer will attach the Floating IP address to the `api-port` automatically.
+
+If you have created the API DNS record, you should be able access the OpenShift API.
+
+##### External Ingress (apps) Access
+
+The installer doesn't currently handle the Ingress floating IP address the same way it does the API one.
+
+To make the OpenShift Ingress access available (this includes logging into the deployed cluster), you will need to attach the Ingress floating IP to the `ingress-port` after the cluster is created.
+
+That can be done in the following steps:
+
+```sh
+openstack port show <cluster name>-<clusterID>-ingress-port
+```
+
+Then attach the FIP to it:
+
+```sh
+openstack floating ip set --port <ingress port id> <apps FIP>
```

Comment:
you can  also use the name here and forget about the previous command: <cluster-name><clusterID>-ingress-port